src/rpc/rpcmapper.rs

```diff
@@ -122,8 +122,8 @@ impl RpcMapper {
     ///
     // TODO This entire thing feels klunky and kludgy; this needs to be revisited...
     pub fn map_response_to_result(response: RpcClientResult) -> RpcPayloadResult {
-        let payload = response?; // Directly returns in case of error
-        Any::try_from(payload)
+        let message = response?; // Directly returns in case of error
+        Any::try_from(message.payload.unwrap())
```

Comment:
Funnily enough, I was doing the same thing [over in this PR](https://github.com/eclipse-uprotocol/up-rust/pull/27) :laughing: 

We should probably be a little more careful here, [similar to what I did on that PR](https://github.com/eclipse-uprotocol/up-rust/pull/27/files#diff-8d66726b55938f3831ad9ba37cb66af6c321e6091a883099cde61b586056cbc3R86) s.t. we don't accidentally panic.

Reply:
Pulled that in - and apologies for waltzing over your work there!